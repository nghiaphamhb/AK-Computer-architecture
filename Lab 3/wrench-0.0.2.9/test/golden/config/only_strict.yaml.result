Config
    { cLimit = 900
    , cMemorySize = 144
    , cInputStreams = Nothing
    , cInputStreamsFlat = Nothing
    , cReports = Just
        [ ReportConf
            { rcName = Just "Executed Instruction Log"
            , rcSlice = AllSlice
            , rcAssert = Nothing
            , rcView = Just "{pc}: {instruction} {pc:label}
              "
            }
        ]
    }
