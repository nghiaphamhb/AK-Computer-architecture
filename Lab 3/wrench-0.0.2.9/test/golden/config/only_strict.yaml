limit: 900
memory_size: 0x90
